streams/readable-streams/brand-checks.js

```diff
@@ -148,4 +148,26 @@ test(() => {
 
 }, 'ReadableStreamController.prototype.error enforces a brand check');
 
+promise_test(t => {
+
+  return Promise.all([
+    methodRejects(t, ReadableStream.prototype, 'cancel', null),
+    methodRejects(t, ReadableStream.prototype, 'cancel', undefined)
+  ]);
+
+}, 'ReadableStream brand checks do not throw on null or undefined');
+
+promise_test(t => {
+
+  return Promise.all([
+    methodRejects(t, ReadableStreamDefaultReader.prototype, 'read', undefined),
+    methodRejects(t, ReadableStreamDefaultReader.prototype, 'read', null)
+  ]);
+
+}, 'ReadableStreamDefaultReader brand checks do not throw on null or undefined');
+
+// There is no way to check ReadableStreamDefaultController brand checks for
```

Comment:
It can't distinguish between buggy implementations and those that are working correctly. In fact, from a standards perspective, "buggy" implementations of IsWritableStreamDefaultController that throw TypeError aren't buggy. Since we're already testing that ReadableStreamDefaultController enforces brand checks, it wouldn't add any value.

We could have a Blink-specific test that checks the exception message, but I don't like doing that, and the value of the test would be small.